lib/plugins/inspec-plugin-manager-cli/lib/inspec-plugin-manager-cli/cli_command.rb

```diff
@@ -20,17 +20,18 @@ def list
         plugin_statuses = Inspec::Plugin::V2::Registry.instance.plugin_statuses
         plugin_statuses.reject! { |s| %i{core bundle}.include?(s.installation_type) } unless options[:all]
 
-        puts
-        ui.bold(format(" %-30s%-10s%-8s%-6s", "Plugin Name", "Version", "Via", "ApiVer"))
-        ui.line
-        plugin_statuses.sort_by(&:name).each do |status|
-          ui.plain(format(" %-30s%-10s%-8s%-6s", status.name,
-                          make_pretty_version(status),
-                          make_pretty_install_type(status),
-                          status.api_generation.to_s))
+        ui.table do |t|
+          t.header = ["Plugin Name", "Version", "Via", "ApiVer"]
+          plugin_statuses.map { |s| [s.name.to_s, s] }.sort { |sa, sb| sa[0] <=> sb[0] }.map { |sf| sf[1] }.each do |status|
```

Comment:
One `sort_by` vs `map`+`sort`+`map`? Not sure I understand this change:
```
plugin_statuses.map { |s| [s.name.to_s, s] }.sort { |sa, sb| sa[0] <=> sb[0] }.map { |sf| sf[1] }.each do |status|
```

vs

```
plugin_statuses.sort_by { |s| s.name.to_s }.each do |status|
```

Reply:
Ahh, of course you can send a block to sort_by. Thank you. Needing to call a chain of methods had me doing the Schwarzian transform on it. Thanks for the suggestion.

Reply:
`sort_by` _is_ the Schwartzian transform pattern.

that and `&:method_name` is `:method_name.to_proc` is `{ |obj| obj.method_name }`